### DISCUSSION TITLE
How to turn off this icon?

### DISCUSSION THREAD

**Discussion by MonfGeiger** [black7375/Firefox-UI-Fix#961] [Q&A] [answered]
I updated firefox, grabbed a new photon ui fix, popped the user.js into it to "update" what settings there were, i had to fiddle around to get _some_ things back to what they were, but i've ran into a......_not quite issue_, but an overall gnat i'd like help getting rid of
![Capture](https://github.com/user-attachments/assets/a859e573-8ad6-477b-8cf4-2df0a9de698d)
basically that blue icon.....how do i turn it off again? i forgor :/

also, i'd like to know if any of my userchrome.css settings are invalid, or if they still work with the ui fix

```
/** Add your custom styles below **********************************************/

/* Enable option to edit bookmark URLs under Add Bookmark (blue star) menu */
/*
 * .editBMPanel_locationRow {
 *   display: initial !important;
 * }
*/

/* Disable Email Image/Audio/Video and Set as Desktop Background context menu items */
/*
 * #context-sendimage,
 * #context-sendvideo,
 * #context-sendaudio,
 * #context-setDesktopBackground {
 *   display: none !important;
 * }
*/

  /* Navbar Border */
  :root:is(:-moz-lwtheme, [lwtheme]) #navigator-toolbox {
    --tabs-border-color: rgba(160, 160, 160, 0.4); /* Legacy: v96, (0, 0, 0, 0.3) -> (0, 0, 0, 0.4)  */
    --lwt-tabs-border-color: rgba(0, 0, 0, 0.4);
  }
```

**Comment by black7375** [comment, marked as answer]
Does this mean you want to remove the blue Insert tab icon?

```css
.tab-drop-indicator {
 display: none; 
}
```

**Reply by MonfGeiger** [reply]
that did it, thanks :)
